docs/reference/commandline/dockerd.md

```diff
@@ -1118,6 +1119,7 @@ This is a full example of the allowed configuration options on Linux:
 	"cluster-advertise": "",
 	"max-concurrent-downloads": 3,
 	"max-concurrent-uploads": 5,
+	"shutdown-timeout": 15,
```

Comment:
It's only supported on Linux, or Windows as well? If so, the "Windows" example needs to be updated as well :smile: